ipaserver/dns_data_management.py

```diff
@@ -139,16 +139,13 @@ def __add_srv_records(
     def __add_ca_records_from_hostname(self, zone_obj, hostname):
         assert isinstance(hostname, DNSName) and hostname.is_absolute()
         r_name = DNSName('ipa-ca') + self.domain_abs
-        rrsets = []
         end_time = time() + CA_RECORDS_DNS_TIMEOUT
         while time() < end_time:
-            try:
-                rrsets = resolve_rrsets(hostname, (rdatatype.A, rdatatype.AAAA))
-            except DNSException:  # logging is done inside resolve_rrsets
-                pass
+            # logging is done inside function
+            rrsets = installutils.resolve_rrsets_nss(hostname)
             if rrsets:
                 break
-            sleep(5)
+            sleep(4)
```

Comment:
Shouldn't the sleep value be a factor of CA_RECORDS_DNS_TIMEOUT?

Reply:
I change sleep to 3sec with 15sec timeout.